src/perfrecord.cpp

```diff
@@ -110,7 +114,7 @@ void PerfRecord::startRecording(bool elevatePrivileges, const QStringList& perfO
 
         // I/O redirection of client scripts launched by kdesu & friends doesn't work, i.e. no data can be read...
         // so instead we use a temporary file and parse its contents via a polling timer :-/
-        auto* outputFile = new QTemporaryFile(m_elevatePrivilegesProcess);
+        auto* outputFile = new QTemporaryFile(this);
```

Comment:
why this change?

Reply:
I can't remember correctly but it was something stupid in hindsight